cdap-watchdog/src/test/resources/rolling-appender-logback-test.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright © 2017 Cask Data, Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  ~ use this file except in compliance with the License. You may obtain a copy of
+  ~ the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations under
+  ~ the License.
+  -->
+
+<configuration>
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>%d{ISO8601} - %-5p [%t:%C{1}@%L] - %m%n</pattern>
+    </encoder>
+  </appender>
+
+  <!--TODO CDAP-8196 Instance Id is provided by logging framework. Remove it after Logging framework is in place-->
+  <property name="cdap.log.saver.instance.id" value="0"/>
+
+  <appender name="rollingAppender" class="co.cask.cdap.logging.plugins.RollingLocationLogAppender">
+    <!--log file path will be created by appender as: <basePath>/namespaceId/applicationId/<filePath>-->
+    <basePath>/tmp/plugins/applogs</basePath>
+    <!--filePath uses instanceId which is a property set by LogFramework. We use it to avoid multiple log saver
+    instances writing to same file in case of kafka partition key reconfiguration supported by CDAP-8104-->
+    <filePath>securityLogs/logFile-${cdap.log.saver.instance.id}.log</filePath>
```

Comment:
Since log framework is not exposing `cdap.log.saver.instance.id`, I am still using this property, but replacing it with program Id in another PR